### PR TITLE
[REF] mail: `NotificationList`, review and simplify scss

### PR DESCRIPTION
**Pull request by xlu-odoo**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2833408

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by xlu-odoo** [comment]
@Brieuc-brd can you please check this PR?
( cc: @stefanorigano )

File: addons/mail/static/src/components/notification_list/notification_list.scss [hunk 1]
```diff
@@ -2,32 +2,10 @@
 // Layout
 // ------------------------------------------------------------------
 
- .o_NotificationList {
-    display: flex;
-    flex-flow: column;
-    overflow: auto;
-
-    &.o-empty {
-        justify-content: center;
```
Comment (Brieuc-brd) [inline addons/mail/static/src/components/notification_list/notification_list.scss:11]: Are you sure you want to remove it ? If not, you can add `justify-content-center` next to `o-empty` in the XML.

File: addons/mail/static/src/components/notification_list/notification_list.xml [hunk 2]
```diff
@@ -39,7 +39,7 @@
                             />
                         </t>
                         <t t-if="!notificationView_last">
-                            <div class="o_NotificationList_separator border-bottom"/>
+                            <div class="o_NotificationList_separator flex-grow-0 flex-shrink-0 border-bottom"/>
```
Comment (Brieuc-brd) [inline addons/mail/static/src/components/notification_list/notification_list.xml:42]: The default value of `flex-grow` is `0`, so it's not necessary to add `flex-grow-0`.

**Review by Brieuc-brd — COMMENTED** [review]
@xlu-odoo 👋 
Some minor remarks 🙂

File: addons/mail/static/src/components/notification_list/notification_list.xml [hunk 3]
```diff
@@ -39,7 +39,7 @@
                             />
                         </t>
                         <t t-if="!notificationView_last">
-                            <div class="o_NotificationList_separator border-bottom"/>
+                            <div class="o_NotificationList_separator w-100 flex-shrink-0 border-bottom"/>
```
Comment (Brieuc-brd) [inline addons/mail/static/src/components/notification_list/notification_list.xml:42]: According to the SCSS coding guidelines, `flex-shrink-0` should be before `w-100`.
https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines

**Comment by Brieuc-brd** [comment]
Ok to me. @xlu-odoo, you can remove the draft state and set your task to « Ready 🟢 ».

@alexkuhn 👋 , this PR is ready for tech. review.

cc. @stefanorigano

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**